pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -35,6 +35,8 @@ in {
   extraMeta ? {},
   # Whether to utilize the controversial import-from-derivation feature to parse the config
   allowImportFromDerivation ? false,
+  # ignored
+  features ? null,
```

Comment:
Excuse me if I'm missing something obvious -- is this change necessary? Am I missing something?

Reply:
`features` has become a mandatory argument for kernels. I don't remember where it is passed as an argument, but without it evaluation fails because some other code expects to be able to pass a `features` argument. I'm not sure where that is, as I said, but this [override](https://github.com/roberth/nixpkgs/blob/b5d54b04b5e2fe3463f14c09226a351f7df9e6d4/pkgs/top-level/all-packages.nix#L13618) supports the idea.
This ignores the `features` parameter entirely, which kind of seems appropriate for a custom kernel. Of course someone could write the code to validate the custom configuration against it and expose it via `passthru` but that's not going to be part of this bugfix.

Reply:
Ah, that seems OK, thanks for the follow up! (I reviewed it a bit 'out of context' since I was busy earlier and hadn't looked deeper). This otherwise looks good to me; I'll merge this (and the backport) soon...